src/pages/waiting-room-to-car/cat-d/waiting-room-to-car.cat-d.page.ts

```diff
@@ -0,0 +1,254 @@
+import { Component, ViewChild } from '@angular/core';
+import { IonicPage, NavController, NavParams, Platform } from 'ionic-angular';
+import { AuthenticationProvider } from '../../../providers/authentication/authentication';
+import { Store, select } from '@ngrx/store';
+import { StoreModel } from '../../../shared/models/store.model';
+import * as waitingRoomToCarActions from '../waiting-room-to-car.actions';
+import { Observable } from 'rxjs/Observable';
+import { getCurrentTest, getJournalData } from '../../../modules/tests/tests.selector';
+import {
+  VehicleRegistrationChanged,
+} from '../../../modules/tests/vehicle-details/common/vehicle-details.actions';
+import { map } from 'rxjs/operators';
+import {
+  InstructorAccompanimentToggled,
+  OtherAccompanimentToggled,
+  SupervisorAccompanimentToggled,
+  InterpreterAccompanimentToggled,
+} from '../../../modules/tests/accompaniment/accompaniment.actions';
+import { getVehicleDetails } from '../../../modules/tests/vehicle-details/cat-d/vehicle-details.cat-d.reducer';
+import { getAccompaniment } from '../../../modules/tests/accompaniment/accompaniment.reducer';
+import {
+  getRegistrationNumber,
+} from '../../../modules/tests/vehicle-details/common/vehicle-details.selector';
+import {
+  getInstructorAccompaniment,
+  getSupervisorAccompaniment,
+  getOtherAccompaniment,
+  getInterpreterAccompaniment,
+} from '../../../modules/tests/accompaniment/accompaniment.selector';
+import { getCandidate } from '../../../modules/tests/journal-data/cat-d/candidate/candidate.cat-d.reducer';
+import { getUntitledCandidateName } from '../../../modules/tests/journal-data/common/candidate/candidate.selector';
+import { getTests } from '../../../modules/tests/tests.reducer';
+import { FormGroup } from '@angular/forms';
+import { QuestionProvider } from '../../../providers/question/question';
+import {
+  TellMeQuestionSelected,
+  TellMeQuestionCorrect,
+  TellMeQuestionDrivingFault,
+  QuestionOutcomes,
+} from '../../../modules/tests/test-data/cat-b/vehicle-checks/vehicle-checks.actions';
```

Comment:
actions are from cat-b

Reply:
These seem to have been copied in from the cat c version of the page. Turns out there is a method that uses these that isn't called. Removed and tested and all is fine. Removed from cat c as well